packages/gasket-plugin-winston/lib/index.js

```diff
@@ -48,7 +48,7 @@ const plugin = {
         ),
         format:
           config.winston?.format ??
-          format.combine(format.splat(), format.json()),
+          format.simple(),
```

Comment:
We would still want to default to `format.combine(format.splat(), format.json())` and only use `format.simple()` when the env is local.

```js
const defaultFormat = gasket.config.env.startsWith('local') ? format.simple() : format.combine(format.splat();
```

```sugggestion
format: config.winston?.format ?? defaultFormat
```